### ISSUE TITLE
Are more complicated directories possible?

### ISSUE THREAD

**Issue by afontenot** [mopidy/mopidy-local-sqlite#89]
About 95% of my music is tagged to Musicbrainz standards, so the Artist tag for individual songs in an album always has all the artists for that song. E.g. "Portugal. The Man feat. Richie Havens & Son Little". This makes browsing by Artist useless, so I added Album Artist as a directory and browse by that most of the time.

It works great, except that I have that other 5% which hasn't been tagged to the same standards and doesn't have an Album Artist tag. Some of these releases aren't on Musicbrainz and it would be a pain to retag them all. Others are classical recordings and should be sorted (IMO) by composer, not the recording artist. 

Basically, it seems to me that local-sqlite should have an "intelligent" directory where it shows the most reasonable thing it can for each album it comes across. E.g. composer for albums with a composer tag, album artist for everything else, and fall back to artist or performer if necessary. 

Apologies if this fits under the "known limitations". If local-sqlite doesn't have this capability, I would be open to helping implement it if there is interest.

**Comment by tkem** [comment]
mopidy-local-sqlite has been in "maintenance-mode" since v1.0.0 (September 2015), so any contributions are more than welcome ;-)
Though I still consider myself to be the "maintainer" of it, meaning I will fix *real* bugs and try to keep  up with any changes to Mopidy core, I haven't used it myself for quite some time (the release of mopidy-dleyna v1.0.0, actually).
So please, if you think you have something to contribute, just go ahead ;-)

**Comment by tkem** [comment]
mopidy-local-sqlite has reach EOL and will be merged into mopidy-local.
